packages/plugins/EVM/src/state/Hub/hub.ts

```diff
@@ -180,6 +190,32 @@ class Hub implements EVM_Hub {
             if (pageable.data.length === 0) return
         }
     }
+    // Alchemy api has different page format
+    getNonFungibleAssetsFromAlchemy(
```

Comment:
Don't add these methods into Hub. Please update the `getNonFungibleAssets` method.